CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_microsoft_ads v0.8.0
+
+*See parallel upstream [release](https://github.com/fivetran/dbt_microsoft_ads_source/releases) of source package.*
```

Comment:
Can you include the specific release of the source package here.

Reply:
Actually, I know it is duplicative but can we include the relevant release notes from the source package here. Just to make it easier for folks to follow along as to why this is being marked as a breaking change.